src/recommendations/release-notes.md

```diff
@@ -0,0 +1,40 @@
+---
+group: product-recommendations
+title: Release Notes
+ee_only: True
+---
+
+The following updates describe the latest improvements to Product Recommendations.
+
+The release notes include:
+
+-  {:.new}New features
+-  {:.fix}Fixes and improvements
+
+## **3.0.2** for Magento Commerce (EE) and Commerce on Cloud (ECE) 2.3.1
+
+-  {:.new}Added a status column on grid when selecting Recommendation units in Page Builder
+-  {:.fix}Fixed an issue with incorrect http/https protocol in product and image URLs
+
+## **3.0.1** for Magento Commerce (EE) and Commerce on Cloud (ECE) 2.3.1
+
+This is a major version release. You will need to [edit your project's root composer.json file](https://devdocs.magento.com/recommendations/install-configure.html#update-your-product-recommendations-installation)
+
+-  {:.new}Page Builder support. With the Page Builder integration, you can accurately and granularly place Recommendation units in any arbitrary location on Page Builder-authored content, as well as style headings and units themselves. This new feature is documented here: [Page Builder support](https://docs.magento.com/user-guide/marketing/page-builder-add-product-recs.html)
+
+-  {:.new}Ability to fetch Product Recommendations from alternate SaaS environments.
```

Comment:
Consider deleting "Ability to" and leave the rest as a sentence fragment, like you did for the first bullet item. Add a brief explanatory sentence, then close with a link to the docs.